### PR TITLE
Clean up Storage node setup

### PR DESCRIPTION
**Pull request by aleitner**
* Edit config on setup
* Default StorageAvailable: 1TiB (https://storjlabs.atlassian.net/browse/V3-1043)
* Default BandwidthAvailable: 500GiB (https://storjlabs.atlassian.net/browse/V3-1043)

*WARNING BACKWARDS INCOMPATIBLE*

### CODE REVIEW

File: pkg/piecestore/psserver/config.go [hunk 1]
```diff
@@ -27,8 +27,8 @@ var (
 // Config contains everything necessary for a server
 type Config struct {
 	Path                   string        `help:"path to store data in" default:"$CONFDIR"`
-	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1GB)" default:"1073741824"`
-	AllocatedBandwidth     int64         `help:"total allocated bandwidth, default(100GB)" default:"107374182400"`
+	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1TiB)" default:"1099511627800"`
```
Comment (egonelbre) [inline pkg/piecestore/psserver/config.go]: maybe use `memory.Size` here instead and implement flag parsing so a user would be able to use `--allocated-disk-space 1TB`

Comment (aleitner) [reply pkg/piecestore/psserver/config.go]: good idea

Comment (aleitner) [reply pkg/piecestore/psserver/config.go]: Using https://github.com/cloudfoundry/bytefmt now

Comment (egonelbre) [reply pkg/piecestore/psserver/config.go]: See https://github.com/storj/storj/blob/master/internal/memory/sizes.go#L8, it should already support it.

Comment (egonelbre) [reply pkg/piecestore/psserver/config.go]: `AllocatedDiskSpace     memory.Size         `help:"total allocated disk space, default(1TiB)" default:"1TB"`, might do it

Comment (aleitner) [reply pkg/piecestore/psserver/config.go]: done

File: go.mod [hunk 2]
```diff
@@ -20,6 +20,7 @@ require (
 exclude gopkg.in/olivere/elastic.v5 v5.0.72 // buggy import, see https://github.com/olivere/elastic/pull/869
 
 require (
+	code.cloudfoundry.org/bytefmt v0.0.0-20180906201452-2aa6f33b730c
```
Comment (egonelbre) [inline go.mod]: this isn't needed anymore now

File: internal/memory/size.go [hunk 3]
```diff
@@ -109,17 +109,17 @@ func (size *Size) Set(s string) error {
 	}
 
 	switch suffix {
-	case "EB":
+	case "EB", "EiB":
```
Comment (egonelbre) [inline internal/memory/size.go]: add tests for it

File: cmd/storagenode/main.go [hunk 4]
```diff
@@ -173,9 +175,30 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
...
+		"log.level":               "debug",
+
+		// "config-dir": "",
+		// "kademlia.bootstrap-addr": "",
+		// "kademlia.external-address": "",
+		// "kademlia.operator.email": "",
+		// "kademlia.operator.wallet": "",
+		// "server.address": "",
+		// "storage.allocated-bandwidth": "",
+		// "storage.allocated-disk-space": "",
```
Comment (jtolds) [inline cmd/storagenode/main.go]: please don't check in commented out code

File: cmd/storagenode/main.go [hunk 5]
```diff
@@ -173,9 +175,21 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		"identity.server.address": defaultServerAddr,
 		"storage.path":            filepath.Join(setupDir, "storage"),
 		"kademlia.bootstrap-addr": defaultSatteliteAddr,
+		"log.level":               "debug",
```
Comment (mniewrzal) [inline cmd/storagenode/main.go]: Why we want to increase to `debug` level?

File: pkg/piecestore/psserver/config.go [hunk 6]
```diff
@@ -28,8 +29,8 @@ var (
 // Config contains everything necessary for a server
 type Config struct {
 	Path                         string        `help:"path to store data in" default:"$CONFDIR"`
-	AllocatedDiskSpace           int64         `help:"total allocated disk space in bytes, default(1GB)" default:"1073741824"`
-	AllocatedBandwidth           int64         `help:"total allocated bandwidth in bytes, default(100GB)" default:"107374182400"`
+	AllocatedDiskSpace           memory.Size   `help:"total allocated disk space in bytes, default(1TiB)" default:"1TiB"`
```
[the remark this answers is not in the document]
Comment (mniewrzal) [reply pkg/piecestore/psserver/config.go]: Also maybe we could add some possible values like 1 GB, 1 TB?

[the remark this answers is not in the document]
Comment (aleitner) [reply pkg/piecestore/psserver/config.go]: done

File: pkg/piecestore/psserver/config.go [hunk 7]
```diff
@@ -28,8 +29,8 @@ var (
...
+	AllocatedBandwidth           memory.Size   `help:"total allocated bandwidth in bytes, default(500GiB)" default:"500GiB"`
```
[the remark this answers is not in the document]
Comment (aleitner) [reply pkg/piecestore/psserver/config.go]: done

File: pkg/piecestore/psserver/config.go (hunk 6 above)
[the remark this answers is not in the document]
Comment (aleitner) [reply pkg/piecestore/psserver/config.go]: 1GB and 1Tb are also usable

File: cmd/storagenode/main.go [hunk 8]
```diff
@@ -94,6 +95,7 @@ func init() {
 	}
 
 	confDir = rootCmd.PersistentFlags().String("config-dir", defaultConfDir, "main directory for storagenode configuration")
+	rootCmd.PersistentFlags().BoolVarP(&editConfig, "edit-conf", "", false, "open config in default editor")
```
[the remark this answers is not in the document]
Comment (aleitner) [reply cmd/storagenode/main.go]: done

File: cmd/storagenode/main.go (hunk 5 above)
Comment (aleitner) [reply cmd/storagenode/main.go]: switched to warn

**Pull request merged**